### ISSUE TITLE
Clarify, whether the policies should be posted word-by-word

### ISSUE THREAD

**Issue by Boldewyn** [EFForg/dnt-policy#23]
This is a request for documentation. https://www.eff.org/dnt-policy says to include the policy _verbatim_ under `.well-known/dnt-policy.txt`. Also, this repo contains a JSON file with hashes for the respective versions.

_However_, the text of the policy contains the sentence

> This file will always be posted via HTTPS at https://example-domain.com/.well-known/dnt-policy.txt
to indicate this fact.

On reading the policy, it seems, that this sentence should be updated to contain the _actual_ domain that issues the policy. In fact, that's how I did it on codepoints.net: https://codepoints.net/.well-known/dnt-policy.txt

Of course, any hashes will be different then. But apart from the above mentioned "verbatim" I found no note, if the file is indeed meant to be included just as downloaded from Github, or is a template (as `https://example-domain.com/` suggests), that needs editing.

The intended way of embedding should be published at the FAQ at https://www.eff.org/dnt-policy, too.

**Comment by baloo** [comment]
I'm facing the same issue, the eff.org itself posted it verbatim (see https://www.eff.org/.well-known/dnt-policy.txt) so I tend to think it should be verbatim. I'd go this way for now.

**Comment by cooperq** [comment]
@baloo is right, the text should be included verbatim. The domain should not be changed to your own domain but left as example-domain.com

**Comment by Boldewyn** [comment]
Yes, I deduced that from various clues. This issue is more meant as a documentation bug.
